BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -63,7 +63,16 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
         self.versionManager = versionManager
         self.goalManager = goalManager
         self.healthStoreManager = healthStoreManager
+        
+        let fetchRequest = Goal.fetchRequest() as! NSFetchRequest<Goal>
+        fetchRequest.sortDescriptors = [NSSortDescriptor(keyPath: \Goal.urgencyKey, ascending: true)]
```

Comment:
For self: is this sometimes the wrong sort?

Reply:
It would have defaulted to urgencyKey then pulled from the user's preference right away. Yes it could have been wrong for a moment initially.
Updated to just use the preference also here.